src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,58 @@
-export const Tabs = () => {};
+import classNames from 'classnames';
+
+interface Tab {
+  id: string;
+  title: string;
+  content: string;
+  index: number;
+}
+
+type Props = {
+  preparedTabs: Tab[];
+  selectedTabIndex: number;
```

Comment:
Pass `selectedTabId` here
```suggestion
  selectedTabId: number;
```